src/lms-ds-loader/edfi_lms_ds_loader/loader_facade.py

```diff
@@ -22,17 +22,37 @@
 logger = logging.getLogger(__name__)
 
 
-@catch_exceptions
+# This module deliberately has no exception handling. Due to the foreign key
+# relationships between tables, there is little point to continuing after a
+# failure. For example: users upload fails. Then section would work, but most of
+# the child-objects of section have a dependency on users. If users succeeds,
+# but section fails, then there's little value to having the users and nothing
+# else. When we get that far down, better to keep the pattern of not catching
+# exceptions throughout then to only catch exceptions on "lower value" targets
+# like grades or attendance.
+
+# TODO: consider refactoring for transaction management, so that operations can
+# be rolled back. SQL Alchemy transactions can be used with DataFrame, so this
+# should be feasible.
```

Comment:
```suggestion
# TODO: consider refactoring for transaction management, so that operations can
# be rolled back. SQL Alchemy transactions can be used with DataFrame, so this
# should be feasible. https://tracker.ed-fi.org/browse/LMS-244
```